test/lib/ansible_test/_data/sanity/code-smell/runtime-metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+    "all_targets": true,
```

Comment:
The test shouldn't need to use `all_targets` since it's only processing the specific files listed below if they exist.

```suggestion
```